### PR TITLE
#2225: fix missing `<cstdint>` include

### LINKED ISSUE #2225: fix missing `<cstdint>` include
**Describe the bug**
`vt` fails to build with `clang 16.0.6` on Fedora 38:
```
/vt/src/vt/configs/error/stack_out.h:54:44: error: use of undeclared identifier 'uint64_t'
using StackTupleType  = std::tuple<int32_t,uint64_t,std::string,std::size_t>;
                                           ^
```

This is because of [changes in Standard Library headers in GCC 13](https://gcc.gnu.org/gcc-13/porting_to.html) (see `Header dependency changes`).

### PR DESCRIPTION
**Pull request by cz4rs**
fixes #2225 

Stumbled upon this when working on NimbleSM :slightly_smiling_face:

### CODE REVIEW

**Review by JacobDomagala — APPROVED** [review]

File: src/vt/configs/error/stack_out.h [hunk 1]
```diff
@@ -44,7 +44,7 @@
 #if !defined INCLUDED_VT_CONFIGS_ERROR_STACK_OUT_H
 #define INCLUDED_VT_CONFIGS_ERROR_STACK_OUT_H
 
-#include <cstdlib>
```
Comment (nmm0) [inline src/vt/configs/error/stack_out.h:47]: I think technically `std::size_t` is from `<cstdlib>` and not `<cstdint>` though likely string and vector will pull it in. But for full portability we should keep this include

File: src/vt/utils/compress/decompressor_base.h [hunk 2]
```diff
@@ -45,7 +45,7 @@
 #define INCLUDED_VT_UTILS_COMPRESS_DECOMPRESSOR_BASE_H
 
 #include <string>
-#include <cstdlib>
```
Comment (nmm0) [inline src/vt/utils/compress/decompressor_base.h:48]: Same as above

**Review by nmm0 — CHANGES_REQUESTED** [review]

**Review by lifflander — APPROVED** [review]

**Comment by cwschilly** [comment]
@cz4rs @lifflander @ppebay I rebased this PR but it is showing Partially Verified again--I think @cz4rs has to do the rebase in order for the commits to be properly signed (see ["Vigilant Mode"](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits)).

**Comment by cz4rs** [comment]
> @cz4rs @lifflander @ppebay I rebased this PR but it is showing Partially Verified again--I think @cz4rs has to do the rebase in order for the commits to be properly signed (see ["Vigilant Mode"](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits)). 

I'll do the rebase in a moment :)

**Pull request merged**